src/Track.cpp

```diff
@@ -372,7 +375,11 @@ bool Track::contains(Point* point) {
         ((point->getY() <= _start.getY()+1.0E-2 &&
           point->getY() >= _end.getY()-1.0E-2)
       || (point->getY() >= _start.getY()-1.0E-2 &&
-          point->getY() <= _end.getY()+1.0E-2)))) {
+          point->getY() <= _end.getY()+1.0E-2)) &&
```

Comment:
This function is actually never used. I believe it was just used in debugging a long time ago. @wbinventor - how do you feel about removing it altogether?